editions/tw5.com/tiddlers/variables/modifier Variable.tid

```diff
@@ -4,7 +4,7 @@ tags: Variables [[Core Variables]]
 title: modifier Variable
 type: text/vnd.tiddlywiki
 
```

Comment:
The modified date should be updated for docs updates.

Reply:
done @Jermolene